tests/test_addition.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from calculator.calculator import Calculator
+
+class TestCalculatorAdditionTests:
+
+    # Test that the add method returns the correct value when passed two or more integers 
+    def test_add_method_returns_correct_result(self):
+        num_list = [5, 5, 5]
```

Comment:
- [x] Add a comment here that references the issue/user story.
- [x] Add user story.